pmlc/dialect/tile/ir/ops.td

```diff
@@ -315,8 +315,8 @@ def GatherOp : SpecialOp<"gather">, HasCanonicalizer {
 
 def PrngOp : SpecialOp<"prng", []>, HasCanonicalizer {
   let summary = "pseudorandom number generator";
-  let arguments = (ins RankedTensorOf<[ScalarUINT32]>:$state, Variadic<EltwiseIndex>:$dims);
-  let results = (outs RankedTensorOf<[AnyScalarFloat]>:$result, RankedTensorOf<[ScalarUINT32]>:$new_state);
+  let arguments = (ins RankedTensorOf<[AnyScalarUnsigned]>:$state, Variadic<EltwiseIndex>:$dims);
+  let results = (outs RankedTensorOf<[AnyScalarFloat]>:$result, RankedTensorOf<[AnyScalarUnsigned]>:$new_state);
```

Comment:
I think this needs to remain a 32-bit value. The lowering for PRNG hasn't been done yet, but the impl in plaidml-v0 was for the state to be UINT32, it's not really up to the user, this is like opaque data that the user shouldn't be concerned with. If there was a make it completely hidden that's what we'd probably want to do.